.circleci/config.yml

```diff
@@ -583,6 +602,8 @@ workflows:
             . venv/bin/activate
             export PATH=${PATH}:${PWD}/tests/integration/bin:/opt/vertica/bin
             export DB_FACTS_PATH=${PWD}/tests/integration/circleci-dbfacts.yml
+            echo $SCRATCH_GCS_URL  # TODO
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker